src/devices/came_top432.c

```diff
@@ -0,0 +1,169 @@
+/** @file
+    Decode CAME remote control TOP-432EV, TOP-432NA, TOP-432EE.
+    This remote control is used for garage door and sliding gate. It transmits on 433.92 MHz (as it is written on the case), built since 2006.
+
+    It works with CAME radio receiver cards "AF43S", capable of handling 4096 codes. CAME is an italian company. Theses remote controls are mainly sold in europ (France, Italy, Belgium). https://www.came.com and https://www.came-europe.com .
```

Comment:
nit: `sold in Europe`

Reply:
Flex decoder can be written with : 
`rtl_433 -r Button1_433.83M_250k.cu8 -X "n=CameTOP432,m=OOK_PWM,s=320,l=640,r=10000,g=830,preamble={1}8"`